README.md

```diff
@@ -4082,8 +4082,13 @@ Does anyone loves archery and choppers the same as I do? :)
 My name is [Kostiantyn](https://github.com/rukhlovka)! And this is my cat and my hands when I was learning Git.
 ![Kitten](./assets/images/cat-Mini.jpg)
 
+
+## Hi! i'm Andre! 
+![Screenshot-image-link](./assets/images/Bandit.jpg) 
+
 ## Hello, Kottans
 My name is Alex.
+<<<<<<< HEAD
```

Comment:
Great work.
The only thing is that this line "<<<<<<< HEAD" should be deleted - it's a part of a "conflict" message.